test/demo/utils/fdc3RemoteExecution.ts

```diff
@@ -327,9 +334,14 @@ function createRemoteIntentListener(executionTarget: Identity, id: number, inten
             }, intent, id);
         },
         getReceivedContexts: async (): Promise<Context[]> => {
-            return ofBrowser.executeOnWindow(executionTarget, function(this: TestWindowContext, intent: IntentType, id: number): Context[] {
-                return this.receivedIntents.filter(entry => entry.listenerID === id && entry.intent === intent).map(entry => entry.context);
-            }, intent, id);
+            if (!listener || !('mock' in listener)) {
```

Comment:
Not properly got my head round this, but feels un-intuitive to a user of this that we'd treat mocks as a special case

Reply:
Removed